### DISCUSSION TITLE
WAI-ARIA compliance issue WCAG 4.1.2

### DISCUSSION THREAD

**Discussion by esginmurat** [orestbida/cookieconsent#542] [General]
Hello thanks for cookieconsent. (@orestbida) WCAG 4.1.2 

I see that the aria-level tag specified in WCAG 4.1.2 is missing. (attached photo)

![aria roles problem](https://github.com/orestbida/cookieconsent/assets/17769013/8f1439a2-f084-4710-a1c2-912c3c2573ea)

Regards...

**Comment by orestbida** [comment]
@esginmurat, nice catch, this has gone unnoticed for so long!

**Reply by esginmurat** [reply]
Sorry.  I did not mean to hurt you.😑
